x/stakeibc/keeper/lsm.go

```diff
@@ -160,24 +160,32 @@ func (k Keeper) GetValidatorFromLSMTokenDenom(denom string, validators []*types.
 
 // Checks if we need to issue an ICQ to check if a validator was slashed
 // The query runs at periodic intervals defined by the ValidatorSlashQueryInterval
-func (k Keeper) ShouldCheckIfValidatorWasSlashed(ctx sdk.Context, validator types.Validator, stakeAmount sdkmath.Int) bool {
+// The interval is represented as percent of TVL
+// (e.g. 1% means every LS that causes the progress to breach 1% of TVL triggers the query)
+func (k Keeper) ShouldCheckIfValidatorWasSlashed(
+	ctx sdk.Context,
+	validator types.Validator,
+	totalHostZoneStake sdkmath.Int,
+	transactionStakeAmount sdkmath.Int,
+) bool {
 	params := k.GetParams(ctx)
-	queryInterval := sdk.NewIntFromUint64(params.ValidatorSlashQueryInterval)
+	queryInterval := sdk.NewDecWithPrec(int64(params.ValidatorSlashQueryInterval), 2) // percentage
```

Comment:
It wasn't clear until I read through the whole PR how this number is supposed to be, even the comment afterwards saying this is a percentage.  If I am understanding right, this is going to be something like 0.01 to indicate every 1% of the total locked will trigger a new check.

Why doing precision 2?  What if we want to change from 1% to 1.5% so we need 0.015 but then won't prec of 2 give us either 0.01 or 0.02?

Reply:
we unfortunately are limited to storing our params as integers right now. Not sure how it got that way, but it was likely something from ignite. It's on by list of future TODOs to allow decimal types

That said, I don't think we need any additional precision and all the other params follow a similar standard for percents so I think it would be confusing if we instead had a param set to "10" which really mean 0.010

Reply:
It's a good callout. I would err on the side of clarity rather than granularity here, since we prob won't need fractional percentage point precision e.g. 1.5%